src/app/blog/_components/archives/theme-switcher.tsx

```
@@ -0,0 +1,113 @@
+  return (
+    <button
+      suppressHydrationWarning
+      className={styles.switch}
+      onClick={handleModeSwitch}
+    />
+  );
+};
+
+const Script = memo(() => (
+  <script
+    dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)